addons/account_peppol/models/account_move_send.py

```diff
@@ -161,13 +161,29 @@ def _call_web_service_after_invoice_pdf_render(self, invoices_data):
                     invoice_data['error'] = _('Please verify partner configuration in partner settings.')
                     continue
 
-                receiver_identification = f"{partner.peppol_eas}:{partner.peppol_endpoint}"
-                params['documents'].append({
-                    'filename': filename,
-                    'receiver': receiver_identification,
-                    'ubl': b64encode(xml_file).decode(),
-                })
-                invoices_data_peppol[invoice] = invoice_data
+                if is_applicable_to_move:
```

Comment:
I prefer 
```suggestion
                if not is_applicable_to_move:
                    continue
```
To spare an unecessary level of indentation. Also, the diff would be a bit better to read (maybe :rofl: who knows ?)